glue/core/visual.py

```diff
@@ -15,20 +15,29 @@
 class VisualAttributes(HasCallbackProperties):
 
     """
-    This class is used to define visual attributes for any kind of objects
-
-    The essential attributes of a VisualAttributes instance are:
-
-    :param color: A matplotlib color string
-    :param alpha: Opacity (0-1)
-    :param linewidth: The linewidth (float or int)
-    :param linestyle: The linestyle (``'solid' | 'dashed' | 'dash-dot' | 'dotted' | 'none'``)
-    :param marker: The matplotlib marker shape (``'o' | 's' | '^' | etc``)
-    :param markersize: The size of the marker (int)
-
+    This class is used to define visual attributes for any kind of objects.
+
+    Parameters
+    ----------
+    parent : :class:`~PyQt5.QtCore.QObject`, optional
+        The object that this visual attributes object is attached to. Default is `None`.
+    color : `str`, optional
+        A matplotlib color string. Default is is set from `~glue.config.SettingsRegistry`.
+    alpha : `float`, optional
+        Opacity, between 0-1. Default is is set from `~glue.config.SettingsRegistry`.
```

Comment:
```suggestion
        A matplotlib color string. Default is is set from `~glue.config.SettingRegistry`.
    alpha : `float`, optional
        Opacity, between 0-1. Default is is set from `~glue.config.SettingRegistry`.
```
typo

Reply:
has this been reverted again?

Reply:
Yeah, I am unsure how I did that.

Reply:
maybe force-pushing from your local branch...

Reply:
I probably forgot to pull and forced pushed without paying attention.